readme.md

```diff
@@ -120,6 +120,10 @@ Turn on/off automatic vendor prefixing (default: `true`)
 - Dynamic styles and themes support
 - CSS Preprocessing via Plugins
 
+## Using in Next.js
+
+Next.js 12+ includes SWC compiler and transformer. You don't have to configure it in babel plugins.
```

Comment:
Even before swc it shouldn't have needed to be configured in babel plugins as Next.js sets it for the user. Might be good to note this. 

```suggestion
Next.js automatically configures `styled-jsx` with babel or swc, you don't have to configure it in babel plugins manually.
```